var/spack/repos/builtin/packages/lorene/local_settings

```diff
@@ -0,0 +1,24 @@
+diff -ruN a/local_settings b/local_settings
+--- a/local_settings	1969-12-31 19:00:00.000000000 -0500
++++ b/local_settings	2021-04-22 16:48:45.000000000 -0400
+@@ -0,0 +1,20 @@
++CXX = g++
```

Comment:
Should this file be templated on the compiler (rather than being a patch), or alternatively should we add conflicts for any compiler other than GCC for the time being?

Reply:
I thought `gcc` etc. were generic names, and that no matter what compiler name you used, you would get Spack's compiler wrapper, which would then call the real compiler (see `lib/spack/env`). Should I use `c++` etc. instead? Or do I need to examine the `spec` object to find the compiler names I need to use?

Reply:
Ok, I get your point. So the details are as follows:
- There is a single compiler wrapper for everything
- When we use it it's through symlinks that have the same name as the wrapped compiler

This permits the wrapper to better disguise as the compiler it's wrapping, since some tools rely on naming for instance. If it is not too much of an effort I think it would be cleaner to turn these patches into "templates" and write them with the appropriate values for `CC` etc. during the `edit` phase of `MakefilePackage. Let me know what you think about that.

Reply:
Done.